schema/rpc.fbs

```diff
@@ -195,6 +195,13 @@ table TapDetectionSettings {
     mounting_reset_delay: float = null;
     mounting_reset_enabled: bool = null;
     mounting_reset_taps: uint8 = null;
+    /// Disables reset behavior of tap detection and sends a TapDetectionSetupResponse,
+    /// each time 2 taps are detected on any tracker
+    setup_mode: bool = null;
+}
+
+table TapDetectionSetupResponse {
```

Comment:
Add comment what this is

Reply:
I also don't see a `TapDetectionSetupRequest`, does that exist?

Reply:
I added a comment on what it actually does, it literally does what it says, there is no extra sauce.

The name comes from what it is used for which is Body Setup, the GUI turns on `setup_mode` for detecting which body part is being tapped, and then assigns the body part with how it normally does it, but it can be used for other stuff if it ever comes the need for it.

Reply:
> I also don't see a `TapDetectionSetupRequest`, does that exist?

It doesn't have one, there is many Responses that dont have a Request, why do you make this assumption?

Reply:
>  there is many Responses that dont have a Request, why do you make this assumption?

Because in request/response, every request has a response, hence the assumption. If we have been violating this, we should stop calling the ones without a request "response". Just because it was done elsewhere doesn't mean that we should keep doing that. We should decide what to call stuff when its basically a server initiated update/notification/event/push. 

Do you have any opinions what to call it? Ideally something that satisfies traditional messaging pattern naming. Later we should go through this and fix/unify the naming of this stuff, but the protocol doesn't really follow true messaging patterns to begin with so its hard qwq.

> Disables reset behavior of tap detection and sends a TapDetectionSetupResponse, each time 2 taps are detected on any tracker

So what happens if this is not set to `null`? This should be documented

> I added a comment on what it actually does, it literally does what it says, there is no extra sauce.

You left a comment on a field in a separate table, so I didn't see the relationship. Just comment on the `TapDetectionSetupResponse` "see `TapDetectionSettings::setup_mode`"

Reply:
> So what happens if this is not set to null? This should be documented
I will say ``Iff true``